### PR TITLE
IA Phase 3

### PR DESCRIPTION
**Pull request by lnhsingh**
Covers Phase 3, which includes major updates in the following sections:

- Connect to CockroachDB
- Schema Design
- Read and Write Data
- Multi-Region Capabilities
- Optimize Performance
- Troubleshooting

Closes DOC-8091

### CODE REVIEW

File: src/current/v23.1/table-localities.md
Comment (nickvigilante) [inline src/current/v23.1/table-localities.md:7]:
```suggestion
key: when-to-use-regional-vs-global-tables.html
---
```

This ensures the version switcher is able to go back and forth between the old page name and new page name.

File: src/current/v23.1/multiregion-survival-goals.md
Comment (nickvigilante) [inline src/current/v23.1/multiregion-survival-goals.md:7]:
```suggestion
key: when-to-use-zone-vs-region-survival-goals.html
---
```

**Review by nickvigilante — CHANGES_REQUESTED** [review]
Great work so far! Two changes to the front matter of your new pages to ensure the version switcher operates as expected. Can you please rebase? Once you do, I'll give my final approval on the PR functionality.

File: src/current/_includes/sidebar-data-v23.1.json
Comment (shannonbradshaw) [inline src/current/_includes/sidebar-data-v23.1.json:12]: I think "connect" should come before "migrate"

File: src/current/v23.1/query-spatial-data.md
Comment (taroface) [inline src/current/v23.1/query-spatial-data.md:95]: Awesome work combining the 2 spatial tutorial docs!

File: src/current/v23.1/transaction-retry-error-reference.md
Comment (taroface) [inline src/current/v23.1/transaction-retry-error-reference.md:60]: I think this example was intentionally put into a separate doc as part of https://github.com/cockroachdb/docs/pull/16280. I wonder if the new IA proposal accidentally excluded that txn retry error example page, and implied that this should be folded back into the txn retry error reference. I don't feel especially strongly about this. What do you think? I'm sorry if I misled you!

**Review by taroface — APPROVED** [review]
LGTM with just a few comments! This is really taking shape.

File: src/current/_includes/v23.1/sidebar-data/schema-design.json
Comment (shannonbradshaw) [inline src/current/_includes/v23.1/sidebar-data/schema-design.json:5]: The page this links to is not an overview of schema design. It's an overview of database schemas. Suggest changing the name to "Database Schemas". 

I'd like to see us include an overview page on CockroachDB Schema Design that includes a treatment of the topics here as part of a later docs project:
https://docs.google.com/document/d/1obWDUJEm1M8cOaLyl5RxXeqBqndtuwJX1I8tjo4a00o/edit#heading=h.yx60tat7xy8

Comment (shannonbradshaw) [reply src/current/_includes/v23.1/sidebar-data/schema-design.json:5]: To be clear, this section of the docs is about schema design, it's just this page that is only about database schemas.

File: src/current/_includes/v23.1/sidebar-data/read-and-write-data.json
Comment (shannonbradshaw) [inline src/current/_includes/v23.1/sidebar-data/read-and-write-data.json:2]: I suggest we change this to "Reads and Writes"

File: src/current/_includes/v23.1/sidebar-data/read-and-write-data.json
Comment (shannonbradshaw) [inline src/current/_includes/v23.1/sidebar-data/read-and-write-data.json:53]: Suggest we put this section before "Write Data" so that it matches the title of the section -- "Read and Write Data"

**Review by shannonbradshaw — APPROVED** [review]
I left a few comments with some suggested changes, but otherwise, LGTM.

File: src/current/v23.1/transaction-retry-error-reference.md
Comment (lnhsingh) [reply src/current/v23.1/transaction-retry-error-reference.md:60]: Added this back!

File: src/current/_includes/v23.1/sidebar-data/schema-design.json
Comment (lnhsingh) [reply src/current/_includes/v23.1/sidebar-data/schema-design.json:5]: Updated the nav title and will file an issue to create a Schema Design Overview page in the future

File: src/current/v23.1/build-a-python-app-with-cockroachdb-asyncpg.md
Comment (nickvigilante) [inline src/current/v23.1/build-a-python-app-with-cockroachdb-asyncpg.md:38]:
```suggestion
To [handle transaction retry errors](query-behavior-troubleshooting.html#transaction-retry-errors), the code uses an application-level retry loop that, in case of error, sleeps before trying the funds transfer again. If it encounters another retry error, it sleeps for a longer interval, implementing [exponential backoff](https://wikipedia.org/wiki/Exponential_backoff).
```

Sorry for this nit, but I updated all Wikipedia links so they could localize themselves when a user accesses the URL.

cc @mdlinville Seems like my new Vale check is working well.

File: src/current/v23.1/build-a-python-app-with-cockroachdb.md
Comment (nickvigilante) [inline src/current/v23.1/build-a-python-app-with-cockroachdb.md:35]:
```suggestion
To [handle transaction retry errors](query-behavior-troubleshooting.html#transaction-retry-errors), the code uses an application-level retry loop that, in case of error, sleeps before trying the funds transfer again. If it encounters another retry error, it sleeps for a longer interval, implementing [exponential backoff](https://wikipedia.org/wiki/Exponential_backoff).
```

File: src/current/v23.1/query-spatial-data.md
Comment (nickvigilante) [inline src/current/v23.1/query-spatial-data.md:19]:
```suggestion
    - The paths between geographic objects are not straight lines; they are curves, and so the distances between objects are calculated using [great circle math](https://wikipedia.org/wiki/Great-circle_distance).
```

**Review by nickvigilante — APPROVED** [review]
Approved if you fix my nits :shipit:

**Pull request merged**
